app/cli/pkg/plugins/manager.go

```diff
@@ -62,7 +63,8 @@ func (m *Manager) LoadPlugins(ctx context.Context) error {
 	for _, plugin := range plugins {
 		// Load the plugin - if there is an error just skip it - we can think of a better strategy later
 		if err := m.loadPlugin(ctx, plugin); err != nil {
-			fmt.Printf("failed to load plugin: %s\n", err)
+			fmt.Printf("failed to load plugin: %s - reason: %s\n", plugin, getFirstErrorLine(err))
```

Comment:
I believe this is not correct, this is poluting stdout, affecting whatever next command output comes in.

I'd use the logger in debug mode (that should send to stderr correct?), and then we can figure out how to add levels?